src/lnd-actions/cancel-invoice.js

```diff
@@ -0,0 +1,20 @@
+const { deadline } = require('../grpc-utils')
+
+/**
+ * Cancel a hold invoice
+ *
+ * @param {string} paymentHash
+ * @param {Object} opts
+ * @param {LndClient} opts.client
+ * @returns {Promise<Object>}
+ */
+function cancelInvoice (paymentHash, { client }) {
+  return new Promise((resolve, reject) => {
+    client.cancelInvoice({ paymentHash }, { deadline: deadline() }, (err, res) => {
```

Comment:
Does this need to be `client.invoices.cancelInvoice`?

Reply:
Yeah, thanks!